### PR TITLE
Feat/support kly qrcode

### PR DESCRIPTION
**Pull request by NikIvv**
Read QR from an image file created in Klayr wallet.

### CODE REVIEW

File: src/components/SendFundsForm.vue [hunk 1]
```diff
@@ -701,13 +703,16 @@ export default {
      */
     onScanQrcode(uri) {
       const recipient = parseURIasAIP(uri)
```
Comment (bludnic) [inline src/components/SendFundsForm.vue:705]: I suggest to create a separate function for parsing KLY based URIs. For example:

```ts
function parseKlyURI(uri: string) {
  // ...
  return { address, amount,  }
}
```

Then you can create a function that will unify the result of both parsers:

```ts
function parseURI() {
  if (isKlyUri) {
    return parseKlyURI()
  }

  return parseURIasAIP() // ADM based URI
}
```

**Review by bludnic — CHANGES_REQUESTED** [review]

**Comment by bludnic** [comment]
[UI: Style "No active ADM nodes" dialog](https://trello.com/c/FKPdukTm/606-ui-style-no-active-adm-nodes-dialog)

**Comment by bludnic** [comment]
[Feat: Enable KLY QR codes from their official app (copied from iOS)](https://trello.com/c/PPllsFpR/603-feat-enable-kly-qr-codes-from-their-official-app-copied-from-ios)

File: src/lib/uri.js [hunk 2]
```diff
@@ -18,11 +20,69 @@ export function getAddressBarURI() {
...
+  return { address, crypto: Cryptos.KLY, params, protocol: KLAYR_WALLET }
+}
+
 /**
  * Parse info from an URI containing a cryptocurrency address
  * Complies with AIP-2, AIP-8, AIP-9
  * Sample: https://msg.adamant.im?address=U9821606738809290000&label=John+Doe&amount=1.12&message=Buy+a+beer
- * @param {string} uri URI. Default is address bar or argv[].
+ * @param {string} URI's origin
+ * @param {string} URI's query parameters
```
Comment (bludnic) [inline src/lib/uri.js:85]: There is only one param `parseURIasAIP(uri)`

File: src/lib/uri.js [hunk 3]
```diff
@@ -18,11 +20,69 @@ export function getAddressBarURI() {
...
+    return key && value
+      ? {
+          ...accum,
+          [key]: window.decodeURIComponent(
+            value.includes('+') ? value.replace(/\+/g, ' ') : value
+          )
+        }
+      : accum
+  }, Object.create(null))
+}
```
[the remark this answers is not in the document]
Comment (NikIvv) [reply src/lib/uri.js:35]: This change, in turn, make us to do further changes (not big ones) in parseURIasAIP(). If we need to get rid of formQueryParamsObject function completely, I'll do that. Please confirm.

[the remark this answers is not in the document]
Comment (bludnic) [reply src/lib/uri.js:35]: You can transform `paramsFromUrl` into an object so it should behave as the previous implementation. Or, is there something else I miss?

File: src/lib/uri.js [hunk 4]
```diff
@@ -74,15 +74,14 @@ function parseKlyURI(query) {
     address = params.recipient || ''
   }
 
-  return { address, crypto: Cryptos.KLY, params, protocol: KLAYR_WALLET }
+  return { address, crypto: Cryptos.KLY, params, protocol: Cryptos.KLY }
```
Comment (bludnic) [inline src/lib/uri.js:77]: Cryptos.KLY -> Cryptos.KLY.toLowerCase()

**Review by bludnic — APPROVED** [review]

**Pull request merged**